source/billing.txt

```diff
@@ -26,14 +26,15 @@ Overview
 --------
 
 Atlas App Services Apps are billed according to how much they're used on a
-monthly basis. There are four independently-measured usage types that determine
-how much you pay each month:
+daily basis. There are four independently-measured usage types that determine
+how much you pay each day:
 
 - :ref:`App Services Requests <billing-requests>`, which measures the number of
   requests that App Services sends and receives from client
   applications and external services. Requests include function calls,
   trigger executions, and sync updates, but exclude user authentication
-  and blocked or invalid requests.
+  and blocked or invalid requests. Pricing for requests is tiered based on
+  how many requests are sent. 
```

Comment:
[Important] We're not doing tiered pricing, right? Should we revert this addition? The only tiers are "Free" and "Not Free", right?

Reply:
Thanks, good catch!